build/test.js

```diff
@@ -67,10 +68,22 @@ function runProtractorTests(doneFn) {
 }
 
 /**
+ * Execute gulp-codecov task and uploads generated
+ * coverage report to http://codecov.io. Should be used only
+ * by external CI tools, as gulp-codecov plugin is already designed to work
+ * with them. Does not work locally.
+ */
+function uploadCoverageReport() {
+  gulp.src(path.join(conf.paths.coverageReport, 'lcov.info')).pipe(codecov());
+}
+
+/**
  * Runs once all unit tests of the application.
  */
 gulp.task('test', ['frontend-test', 'backend-test']);
 
+gulp.task('coverage-codecov-upload', uploadCoverageReport);
```

Comment:
Can you inline the uploadCoverageReport function here? And move the documentation also here.